pkg/oauth/api/types.go

```diff
@@ -144,6 +144,8 @@ type OAuthClientAuthorization struct {
 	Scopes []string
 }
 
+type SelfOAuthClientAuthorization OAuthClientAuthorization
```

Comment:
I think I'd expect a different type that omits the UserName/UserUID fields... we need to explicitly convert between the two types anyway to make watch work, and to verify/adjust the name)